Bio/PDB/DSSP.py

```diff
@@ -38,29 +38,34 @@
 _dssp_cys = re.compile('[a-z]')
 
 # Maximal ASA of amino acids
-# Values from Sander & Rost, (1994), Proteins, 20:216-226
 # Used for relative accessibility
-MAX_ACC = {}
-MAX_ACC["ALA"] = 106.0
-MAX_ACC["CYS"] = 135.0
-MAX_ACC["ASP"] = 163.0
-MAX_ACC["GLU"] = 194.0
-MAX_ACC["PHE"] = 197.0
-MAX_ACC["GLY"] = 84.0
-MAX_ACC["HIS"] = 184.0
-MAX_ACC["ILE"] = 169.0
-MAX_ACC["LYS"] = 205.0
-MAX_ACC["LEU"] = 164.0
-MAX_ACC["MET"] = 188.0
-MAX_ACC["ASN"] = 157.0
-MAX_ACC["PRO"] = 136.0
-MAX_ACC["GLN"] = 198.0
-MAX_ACC["ARG"] = 248.0
-MAX_ACC["SER"] = 130.0
-MAX_ACC["THR"] = 142.0
-MAX_ACC["VAL"] = 142.0
-MAX_ACC["TRP"] = 227.0
-MAX_ACC["TYR"] = 222.0
+
+residue_max_acc = dict(
+    # Miller max acc: Miller et al. 1987 (http://dx.doi.org/10.1016/0022-2836(87)90038-6)
+    # Wilke: Tien et al., 2013 (http://dx.doi.org/10.1371/journal.pone.0080635)
+    # Sander: Sander & Rost, (1994), Proteins, 20:216-226
+    Miller={
```

Comment:
Clearly you are not used to using the dict() constructor:
https://docs.python.org/3/library/stdtypes.html#dict

I imagine that you would rather have something like this:
```
residue_max_acc = {
    # This dict is taken from the Wilke Lab's DSSP wrapper.
    # The code for the wrapper can be found at: http://wilke.openwetware.org/Software.html ("Program for computing relative solvent accessibilities for a protein that uses the DSSP program of Kabsch and Sander")
    # The values in the dict are presented in Tien et al., 2013 (http://dx.doi.org/10.1371/journal.pone.0080635)
    # and seem to be taken from the proposed values for ASA normalization (in A^2) from Miller et al., 1987 (http://dx.doi.org/10.1016/0022-2836(87)90038-6).
    'Miller': {
        'A': 113.0, 'R': 241.0, 'N': 158.0, 'D': 151.0,
        'C': 140.0, 'Q': 189.0, 'E': 183.0, 'G': 85.0,
        'H': 194.0, 'I': 182.0, 'L': 180.0, 'K': 211.0,
        'M': 204.0, 'F': 218.0, 'P': 143.0, 'S': 122.0,
        'T': 146.0, 'W': 259.0, 'Y': 229.0, 'V': 160.0
    },
    'Wilke': {
        'A': 129.0, 'R': 274.0, 'N': 195.0, 'D': 193.0,
        'C': 167.0, 'Q': 225.0, 'E': 223.0, 'G': 104.0,
        'H': 224.0, 'I': 197.0, 'L': 201.0, 'K': 236.0,
        'M': 224.0, 'F': 240.0, 'P': 159.0, 'S': 155.0,
        'T': 172.0, 'W': 285.0, 'Y': 263.0, 'V': 174.0
    },
}
```

Which is fine with me, as long as the ASA values get updated - one way or the other. And btw the code runs fine and it would indeed have failed a unit test if I had used an unhashable type as keys, but that just isn't the case.